SUBMITTED-PLUGINS.md

```diff
@@ -21,6 +21,8 @@
 
 - llllvvuu/nvim-js-actions
 - julienvincent/nvim-paredit
+- bennypowers/nvim-regexplainer
```

Comment:
No need to add `nvim-regexplainer` and `splitjoin.nvim` here since they are already tracked on Dotfyle
- https://dotfyle.com/plugins/bennypowers/nvim-regexplainer
- https://dotfyle.com/plugins/bennypowers/splitjoin.nvim

Reply:
If you want to submit the regexplainer update for twin, i've taken a note of the post of your link and will try to include it in the next issue.